api/getMovieDetails/index.mjs

```diff
@@ -4,6 +4,7 @@ const { API_BASE_MOVIES_URL, API_KEY, API_LANGUAGE } = process.env;
 
 export default async function getMovieDetails(req, res) {
   if (req.method === 'POST') {
+    // To test POST request with _getMovieDetails.http use: const { id } = req.body;
```

Comment:
Is this the right place for this comment? Also, the comment does not really tell me what to do exactly. Maybe rephrase it or leave it out? Your code is quite understandable.